.travis.yml

```diff
@@ -4,7 +4,8 @@ python: 3.7
 cache: pip
 branches:
   only:
-  - master
+  - develop
+  - prod
```

Comment:
I'd prefer that the split be between master and all other branches, but iirc you had a reasons why you don't like doing that. Can you remind me?

Reply:
It may have been to cut down on the number of builds.

Reply:
it's just a convention that was started by the Bridge team.  I've got files like install_aws_cli.sh that has this convention built it can be reused here.  Actually this convention is used by many Sage projects now.